style.css

```diff
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  background-color: #e5e5e5;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+.header-nav {
+  display: flex;
+  height: 60px;
+  background-color: #fff;
+  justify-content: center;
+  justify-content: space-between;
+  align-items: center;
+  padding: 50px 36px 15px 24px;
+  box-sizing: border-box;
+}
+
+#mylogo {
+  width: 76px;
+  height: 20px;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  transition: transform 0.4s ease-in-out;
+}
+
+#logo:hover {
+  transform: scale(1.1);
+}
+
+#menu {
+  display: block;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #6070ff;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+#nav {
+  display: none;
+}
+
+#link {
+  display: none;
+}
+
+.flex-container {
+  height: 700px;

```

Comment:
- Hey, let's kindly remove this fixed height and instead add a padding-bottom: from 70px. Play with the values to get something closer to the length as seen in the Figma design.